setup.cfg

```
@@ -4,7 +4,7 @@ commit = True
 tag = True
 
 [metadata]
-description-file = README.md
+description_file = README.md
```

```suggestion
description-file = README.md
```
this was correct before, to add the description to PyPI

---

When testing the schema installs with setup.py, I got a warning saying the dash is deprecated, and to use underscore.

---

```
(venv) mike@MSI:~/workspace/nanome-lib$ python setup.py install 
/home/mike/workspace/nanome-lib/venv/lib/python3.7/site-packages/setuptools/dist.py:760: UserWarning: Usage of dash-separated 'description-file' will not be supported in future versions. Please use the underscore name 'description_file' instead
```